maint/local_python/binding_c.py

```diff
@@ -1801,6 +1801,8 @@ def dump_validation(func, t):
         if RE.match(r'mpi_(i?m?probe|i?recv|recv_init)$', func_name, re.IGNORECASE) or name == "recvtag":
             # allow MPI_ANY_SOURCE, MPI_PROC_NULL
             G.out.append("MPIR_ERRTEST_RECV_RANK(%s, %s, mpi_errno);" % (comm_ptr, name))
+        elif RE.match(r'mpi_(isendrecv)', func_name, re.IGNORECASE) and name == "source":
```

Comment:
This won't catch `MPI_Sendrecv` or `MPI_Sendrecv_replace`, will it? Should it be something like `r'mpix?_(i?sendrecv)'`?

Reply:
Yes. I don't what I was thinking :) 

Fixed.

We don't need `mpix?_` because the `mpix` prefix has not been added here yet (it is added in `get_function_name`).